pkg/issuer/acme/dns/dns.go

```diff
@@ -262,6 +264,24 @@ func (s *Solver) solverForIssuerProvider(issuer v1alpha1.GenericIssuer, provider
 			accountSecretBytes,
 			s.DNS01Nameservers,
 		)
+	case providerConfig.DNSMadeEasy != nil:
+		secretKey, err := s.secretLister.Secrets(resourceNamespace).Get(providerConfig.DNSMadeEasy.SecretKey.Name)
+		if err != nil {
+			return nil, fmt.Errorf("error getting dnsmadeeasy secret key: %s", err)
+		}
+
+		secretKeyBytes, ok := secretKey.Data[providerConfig.DNSMadeEasy.SecretKey.Key]
+		if !ok {
+			return nil, fmt.Errorf("error getting dnsmadeeasy secret key: key '%s' not found in secret", providerConfig.DNSMadeEasy.SecretKey.Key)
+		}
+
+		impl, err = s.dnsProviderConstructors.dnsMadeEasy(
+			providerConfig.DNSMadeEasy.BaseURL,
+			providerConfig.DNSMadeEasy.APIKey,
+			string(secretKeyBytes),
+			providerConfig.DNSMadeEasy.DomainID,
+			s.DNS01Nameservers,
+		)
```

Comment:
Please add tests for config loading in `dns_test.go` (similar to other providers).

We should check all cases, i.e. verify that the code behaves as expected when malformed configuration is provided *as well as* for valid configuration.